README.md

```diff
@@ -1,5 +1,5 @@
-# NWNX:EE (build 8186) ([Changelog](CHANGELOG.md))
-
+# NWNX:EE (build 8186) {#mainpage}
+[TOC]
```

Comment:
Doesn't this make the github page all weird?

Reply:
For now, yes. I wouldn't exactly say "all weird" but those tags are rendered. With that said reworking the intro README.ms is next on the list. I would like to separate building, installing, contributing and docker images into their own files and at that point we'll make sure it renders nicely on GitHub and in our docs.